locale/en/blog/release/v8.0.0.md

```diff
@@ -0,0 +1,1216 @@
+
+### Say hello to async_hooks
+
+The experimental `async_hooks` module (formerly `async_wrap`) has received a
+major upgrade in 8.0.0. This diagnostics API allows developers a means of
+monitoring the operation of the Node.js event loop, tracking asynchronous
+requests and handles through their complete lifecycle.
+[Complete documentation](https://github.com/nodejs/node/pull/13287) for the
+new module is still incomplete and users should take great care when using the
+experimental new module.
+
+### Say hello to the WHATWG URL parser
+
+An experimental `URL` API implemented around the
+[WHATWG URL Standard](https://url.spec.whatwg.org/) was added to Node.js 7.x
+last year and has been under active development ever since. We are excited to
+announce that as of 8.0.0, the new `URL` implementation is now a fully
+supported, non-experimental API within Node.js. An example usage is shown below,
+with more details available in the
+[official documentation](https://nodejs.org/api/url.html#url_the_whatwg_url_api).
+
+```js
+const URL = require('url').URL;
+
+const myUrl = new URL('/a/path', 'https://example.org/');
+```
+
+This new `URL` implementation is most significant in that it matches the `URL`
+implementation and API available in modern Web Browsers like Chrome, Firefox,
+Edge, and Safari, allowing code using URLs to be shared across environments.
+
+### Buffer changes
+
+A number of important changes have been made to the `Buffer` API within Node.js.
+Most significant is the fact that calling the deprecated `Buffer(num)`
+constructor (with or without the `new` keyword) will return a zero-filled
+`Buffer` instance. Prior versions of Node.js would return uninitialized memory,
+which could contain potentially sensitive data.
+
+In Node.js 6.0.0, a new set of `Buffer` construction methods
+[were introduced](https://medium.com/@jasnell/node-js-buffer-api-changes-3c21f1048f97)
+as an alternative to calling the `Buffer(num)` constructor in order to address a
+number of [security](https://nodesecurity.io/advisories/67) and usability
+concerns. The existing constructor, however, is used extensively throughout the
+Node.js ecosystem, making it infeasible for us to fully deprecate or disable it
+without causing significant breakage.
+
+Zero-filling new instances of `Buffer(num)` by default will have a noticeable
+and significant impact on application performance. To mitigate the loss in
+performance, developers should move to the new `Buffer.allocUnsafe(num)` API if
+they wish to allocate `Buffer` instances with uninitialized memory. Examples of
+zero-filled and uninitialized `Buffer` creation in Node.js 8 are shown below.
+
+```js
+// Zero-filled Buffers
+const safeBuffer1 = Buffer.alloc(10);
+const safeBuffer2 = new Buffer(10);
+
+// Uninitialized Buffer
+const unsafeBuffer = Buffer.allocUnsafe(10);
+```
+
+Note that while the `Buffer(num)` constructor will not be removed from Node.js,
+its continued use is considered to be unsupported.
```

Comment:
At a minimum: `considered to be unsupported` -> `deprecated`

But really, I'd say remove this sentence altogether. We can't truly guarantee that it will not be removed from Node.js ever (although I agree it's unlikely).